Update Book Dash chapter to clarify who our target audience is (#2303)
alan-turing-institute/the-turing-way: book/website/community-handbook/bookdash.md

In the past, we have organised 1-2 days long Book Dash events in person or in a hybrid format, where one of the participants coordinated with their team remotely. 
However, to ensure that international participants have an equal chance to join and address the challenges of hosting in-person events during the COVID-19 pandemic, we started hosting Book Dashes virtually. 
These virtual Book Dashes are five days long, designed for flexible participation by members in different time zones. 

> Meaning, rather than committing their entire working day, participants can choose one or multiple 2.5 hours short collaborative co-working calls, called contribution sessions each day based on their availability.

Reviewer: ```suggestion
Meaning, rather than committing their entire working day, participants can choose one or multiple 2.5 hours short collaborative co-working calls, called "contribution sessions" each day based on their availability.
```
Just adding this suggestion, if this is what they are officially called :)

Author: Good catch Anne.

> In the future, we would like to co-design hybrid events with "virtual-first" approach for the Book Dash and additional opportunity for participants to organise small local meet-ups for collaboration and social events.

Author: ```suggestion
In the future, we will co-design hybrid events with "online-first" approach for the Book Dash attendees who will have the opportunity to organise small local meet-ups for collaboration and social events.
```
